image_detector/BallPosition[] balls
